src/sentry/testutils/cases.py

```diff
@@ -1287,7 +1286,17 @@ def store_segment(
 
         transaction = transaction or "/hello"
 
-        payload: Span = {
```

Comment:
`Span` is now `CompatibleSpan`, which does not make sense as a type here. I had to move up the assignments to `sentry_tags` to keep mypy happy.